etc/README.md

```diff
@@ -0,0 +1,46 @@
+# Installation instructions
+
+## bash
+
+Open your `.bashrc` file and add:
+
+> If you want to set up Git to automatically have Bash shell completion for all users, copy the `hub.bash_completion` script to the `/opt/local/etc/bash_completion.d` directory on Mac systems or to the `/etc/bash_completion.d/` directory on Linux systems. ([Source](https://git-scm.com/book/en/v1/Git-Basics-Tips-and-Tricks#Auto-Completion))
```

Comment:
Why is this marked up as a blockquote?

Also note that `/opt/local/etc/bash_completion.d` directory doesn't exist by default on OS X. This is the directory which is used for bash completion scripts by Homebrew if you installed it in `/opt/local`, which is minority of users. Most of them will have Homebrew in `/usr/local`. However if they use Homerew, then they can just use `brew install hub` and they won't have to do any extra steps to set up bash completion. So I guess that Homebrew needs its own mention.